src/callbacks.py

```diff
@@ -17,34 +17,34 @@ def __init__(self, image_nr, image_resize, model_name):
 
     def on_epoch_end(self, *args, **kwargs):
         self._send_numeric_channels()
-        self._send_image_channels()
+        # self._send_image_channels()
         self.epoch_id += 1
 
-    def _send_image_channels(self):
-        self.model.eval()
-        pred_masks = self.get_prediction_masks()
-        self.model.train()
-
-        for name, pred_mask in pred_masks.items():
-            for i, image_duplet in enumerate(pred_mask):
-                h, w = image_duplet.shape[1:]
-                image_glued = np.zeros((h, 2 * w + 10))
-
-                image_glued[:, :w] = image_duplet[0, :, :]
-                image_glued[:, (w + 10):] = image_duplet[1, :, :]
-
-                pill_image = Image.fromarray((image_glued * 255.).astype(np.uint8))
-                h_, w_ = image_glued.shape
-                pill_image = pill_image.resize((int(self.image_resize * w_), int(self.image_resize * h_)),
-                                               Image.ANTIALIAS)
-
-                self.ctx.channel_send('{} {}'.format(self.model_name, name), neptune.Image(
-                    name='epoch{}_batch{}_idx{}'.format(self.epoch_id, self.batch_id, i),
-                    description="true and prediction masks",
-                    data=pill_image))
-
-                if i == self.image_nr:
-                    break
+    # def _send_image_channels(self):
+    #     self.model.eval()
+    #     pred_masks = self.get_prediction_masks()
+    #     self.model.train()
+    #
+    #     for name, pred_mask in pred_masks.items():
+    #         for i, image_duplet in enumerate(pred_mask):
+    #             h, w = image_duplet.shape[1:]
+    #             image_glued = np.zeros((h, 2 * w + 10))
```

Comment:
@taraspiotr this doesn't need to be commented out as long as the `self._send_image_channels()` is